modules/backend/widgets/form/assets/js/october.form.js

```diff
@@ -137,6 +141,10 @@
             formEl = this.$form,
             fieldElements = this.getFieldElements()
 
+        if (!form) {
+            return;
+        }
```

Comment:
Can you explain under what conditions this triggers?

Reply:
When adding item, then removing one, then adding one again. The form element cannot be found. Don't know why yet

Reply:
It fixed this
![CleanShot 2019-11-05 at 22 32 33](https://user-images.githubusercontent.com/6506510/68248000-454d4e80-001c-11ea-811b-cd35ff5cd942.gif)